### DISCUSSION TITLE
TP4 - Enmienda al ejercicio 9 y 11.

### DISCUSSION THREAD

**Discussion by Mirkopoj** [INGCOM-UNRN-P1/cursada-2023#129] [Preguntas y respuestas] [answered]
El titulo del ejercicio no está relacionado con la consigna. No hay ninguna duda, solo aviso por si hubo algun error.

**Comment by martinvilu** [comment]
¿Qué cosa?

**Reply by ELVAVAS** [reply]
El ejercicio se llama "Elimina duplicados" y pide que inviertas los términos de un arreglo... No hay relación entre el nombre y el ejercicio

**Reply by martinvilu** [reply]
Ok, sean claros entonces; ahora lo reviso

**Reply by ELVAVAS** [reply]
Sí sí, acabo de llegar a ese ejercicio y vine a ver si alguien hablaba de eso discussions.

**Comment by martinvilu** [comment, marked as answer]
Corrección en el enunciado

### Ejercicio 7 - Largo seguro II
Encontrar la longitud de una cadena segura empleando aritmética de punteros y sin usar el operador de arreglos `[]`.

```C
int largo_seguro(char *str, int capacidad_cadena);
```

### Ejercicio 9 - Elimina duplicados

Implementar una función que elimine los duplicados de un arreglo, retornando el nuevo tamaño.

### Ejercicio 10 - Reemplazo

Implementar una función que reemplace las vocales de una cadena segura reemplazándolas por un carácter indicado.

### Ejercicio 11 - Inversión

Invertir el orden de los elementos presentes en un arreglo
```c
int invertir(int arreglo[], int tamaño);
```

Lo corregí en el repositorio, pero modificarles la consigna a cada uno es tentar a los `git merge`.

Reemplacen los últimos ejercicios con estos.
